test/cypress/integration/30-archiveCollective.test.js

```diff
@@ -1,39 +1,52 @@
 describe('Archive Collective', () => {
-  it('Should archive organization', () => {
-    cy.login().then(() => {
-      // Create a new organization
-      cy.createCollective({ type: 'ORGANIZATION' }).then(collective => {
-        const collectiveSlug = collective.slug;
-        cy.visit(`/${collectiveSlug}/edit/advanced`);
-        cy.contains('button', 'Archive this Organization').click();
-        cy.get('[data-cy=action]').click();
-        cy.wait(500);
-        cy.contains('This organization has been archived');
-      });
+  let orgSlug = null;
+  let collectiveSlug = null;
+
+  before(() => {
+    cy.createCollective({ type: 'ORGANIZATION' }).then(({ slug }) => {
+      orgSlug = slug;
+    });
+  });
+
+  before(() => {
+    cy.createCollective({ type: 'COLLECTIVE' }).then(({ slug }) => {
+      collectiveSlug = slug;
+    });
+  });
```

Comment:
:ice_cream: maybe simpler to put everything in a single `before`?

```suggestion
  before(() => {
    cy.createCollective({ type: 'ORGANIZATION' }).then(({ slug }) => {
      orgSlug = slug;
    });
    
    cy.createCollective({ type: 'COLLECTIVE' }).then(({ slug }) => {
      collectiveSlug = slug;
    });
  });
```

Reply:
Ahhhh right sorry, sometimes I forget you don't have to separate the befores for each particular test 🤦‍♀️